content/pages/joinus.md

```diff
@@ -0,0 +1,103 @@
+---
+title: شارك معنا 
+slug: joinus
```

Comment:
i would name the slug something more readable like join-us

Reply:
done